llm_client/llm_api_client/huggingface_client.py

```diff
@@ -0,0 +1,42 @@
+from llm_client.llm_api_client.base_llm_api_client import BaseLLMAPIClient, LLMAPIClientConfig
+from transformers import AutoTokenizer
+
+DEFAULT_DIR = "OpenAssistant"
+BASE_URL = f"https://api-inference.huggingface.co/models/{DEFAULT_DIR}/"
+COMPLETIONS_KEY = 0
+INPUT_KEY = "inputs"
+TEXT_KEY = "generated_text"
+AUTH_HEADER = "Authorization"
+BEARER_TOKEN = "Bearer "
+DEFAULT_MODEL = "oasst-sft-4-pythia-12b-epoch-3.5"
+CONST_SLASH = '/'
+EMPTY_STR = ''
+NEWLINE = '\n'
+
+
+class HuggingFaceClient(BaseLLMAPIClient):
```

Comment:
can you add use of HuggingFaceClient in examples/only_llm_api_client.py

Reply:
will do